src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py

```diff
@@ -118,11 +118,11 @@ def test_none_dist_strat_multi_worker_strategy(self):
     def test_none_dist_strat_mwms_custom_img(self):
         tfc.run(
             entry_point=os.path.join(self.test_data_path,
-                                     "mnist_example_using_ctl.py"),
-            worker_count=2,
+                                     "mnist_example_using_fit.py"),
+            distribution_strategy="auto",
             requirements_txt=os.path.join(self.test_data_path,
                                           "requirements.txt"),
-            docker_base_image="tensorflow/tensorflow:latest-gpu",
+            docker_base_image="gcr.io/deeplearning-platform-release/tf2-gpu.2-2:latest",
```

Comment:
do you need to update version matching check with local environment?

Reply:
No, in this use case a local TF installation is not required and shouldn't affect the image TF version.

Reply:
I meant the previous comment in a general case... The user code `mnist_example_using_fit.py` may have been written in a particular set of API symbols that may be before or past TF2.2. Because of the general backward comparability of TF symbols between minor versions, it'd be generally safe to try to use latest, but wasn't sure if it'd be safe to pin on 2.2

Reply:
Actually, because of the comment I made above re; maybe assumptions made on tagging scheme, I may recommend updating `docker_base_image` in a separate PR, to isolate changes.

Reply:
+1 It would be great if you could separate out the `docker_base_image` PR. There are some nuances here that might be better to follow up on separately.

Reply:
The docker_base_image name has been updated here just to showcase that non docker hub images can be used. We can ofcourse change the unit test to use a different gcr.io image.

Reply:
Removed the nightly version check.

Reply:
Maybe you should revert this base image in the integration test, to contain one change at a time.

Reply:
I removed the check for seeing whether an image is in docker hub in this PR. This is for testing that. it's unrelated to the nightly version change.